mobile/src/navigations/onboarding-navigation/onboarding-navigation.tsx

```diff
@@ -17,15 +26,38 @@ const OnboardingNavigation: React.FC = () => {
                 header: ({ navigation }): React.ReactNode => (
                     <Header navigation={navigation} />
                 ),
+                drawerStyle: {
+                    width: 330,
+                },
             }}
+            drawerContent={(
+                props: DrawerContentComponentProps,
+            ): React.ReactNode => <Steps {...props} />}
         >
             <Drawer.Screen
                 name={OnboardingScreenName.PROFILE}
                 component={Profile}
+                initialParams={{ isCompleted: false, disabled: false }}
             />
             <Drawer.Screen
                 name={OnboardingScreenName.BSA_BADGES}
                 component={BsaBadges}
+                initialParams={{ isCompleted: false, disabled: false }}
```

Comment:
we can access this initialParams data in steps component in Drawer, is it correct?

Reply:
Yes, we can pass to the screen prop 'route' and via 'route.params' we will be able to change this params. Like after completing form we will change isCompleted to 'true'